Registry server: config tests fail when run with `make test` and nonempty `APG_REGISTRY` variables

After setting the auth variables with `source auth/LOCAL.sh` and running `make test`, I get the following test failures:

```
--- FAIL: TestClientBadConfig (0.00s)
    client_test.go:27: expected error
    client_test.go:37: expected error
    client_test.go:46: expected error
--- FAIL: TestAllConfigs (0.06s)
    config_test.go:299: unexpected diff: (-want +got):
          map[string]connection.Config{
                "config1": {Address: "localhost:8080", Insecure: true},
                "config2": {
        -               Address:  "remote:8888",
        +               Address:  "localhost:8080",
        -               Insecure: false,
        +               Insecure: true,
                        Token:    "",
                },
          }
FAIL
```

--- a/pkg/config/configuration_test.go
+++ b/pkg/config/configuration_test.go
@@ -281,6 +281,8 @@ func TestManipulations(t *testing.T) {
 
 func TestAllConfigs(t *testing.T) {
 	t.Cleanup(test.CleanConfigDir(t))
+	t.Setenv("APG_REGISTRY_ADDRESS", "")
+	t.Setenv("APG_REGISTRY_INSECURE", "")
 
 	config1 := config.Configuration{
 		Registry: config.Registry{

--- a/pkg/connection/client_test.go
+++ b/pkg/connection/client_test.go
@@ -23,6 +23,8 @@ import (
 
 func TestClientBadConfig(t *testing.T) {
 	t.Cleanup(test.CleanConfigDir(t))
+	t.Setenv("APG_REGISTRY_ADDRESS", "")
+	t.Setenv("APG_REGISTRY_INSECURE", "")
 
 	_, err := NewRegistryClient(context.Background())
 	if err == nil {

